googlepubsub/README.md

```diff
@@ -0,0 +1,94 @@
+# Google Cloud Pub/Sub Bindings
+
+This document defines how to describe Google Cloud Pub/Sub specific information with AsyncAPI.
+
+<a name="version"></a>
+
+## Version
+
+Current version is `1.0.0`.
```

Comment:
you want first version to be major?

Reply:
I wasn't sure what to use here, just not sure what else I see changing after we agree with its implementation.  I'm all ears for suggestions.

Reply:
For other bindings, first public revision is usually marked as `0.1.0`. I think we can stick with convention.

Reply:
I definitely want to stick with convention, so I'll update the PR and resolve this when it's done.